packages/teleport/src/Kubes/ConnectDialog/ConnectDialog.story.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import ConnectDialog from './ConnectDialog';
+
+export default {
+  title: 'Teleport/Kubes/ConnectDialog',
```

Comment:
```suggestion
  title: 'Teleport/Kubes',
```
doesn't need another directory b/c its self contained